.travis.yml

```diff
@@ -1,9 +1,9 @@
 language: python
 before_install:
  - sudo apt-get update -qq
- - sudo apt-get install -qq mpg123 portaudio19-dev libglib2.0-dev swig bison libtool autoconf libglib2.0-dev libicu-dev libfann-dev
+ - sudo apt-get install -qq mpg123 portaudio19-dev libglib2.0-dev swig bison libtool autoconf libglib2.0-dev libicu-dev libfann-dev realpath
 python:
-  - "2.7"
+  - "3.4"
```

Comment:
Could this be tested against multiple py3 versions? Like here: https://github.com/alexa-pi/AlexaPi/blob/master/.travis.yml

Reply:
Much easier than anticipated :)